design/baremetal-operator/unmanaged-state.md

```diff
@@ -0,0 +1,139 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# unmanaged-state
+
+## Status
+
+implementable
+
+## Summary
+
+This design proposes adding a new state to the state machine to be
+applied to hosts that are known but cannot be managed because the BMC
+credentials are not.
+
+## Motivation
+
+As a precursor to being able to provision hosts without their BMC
+credentials, we want to be able to define hosts without BMC
+credentials so that the credentials can be added at a later
+time. Consumers of the hosts, including the
+`cluster-api-provider-metal3`, need to be able to recognize such hosts
```

Comment:
Should we also include `cluster-api-provider-baremetal` here now that the projects differ significantly.

Reply:
I left that out because it's a downstream concern, but the changes would essentially be the same.